gulpfile.js

```diff
@@ -23,5 +24,10 @@ gulp.task('copydecs', () => {
 	.pipe(gulp.dest(tsProject.options.declarationDir));
 });
 
-gulp.task('build', [ 'typescript', 'copydecs' ]);
+gulp.task('watch', () => {
+	watch('./lib', () => {
```

Comment:
We may need to be careful about multiple directories being present (e.g. both `./lib/` and  `./typings/`) but sounds good :+1: